layouts/partials/head.html

```diff
@@ -8,6 +8,16 @@
     name="description"
     content="{{if .IsHome}}{{$data.description | default $.Site.Data.config.description}}{{else}}{{.Summary}}{{end}}"
   />
+  <meta property="og:image" content="https://quartz.jzhao.xyz/icon.png">
```

Comment:
Could you make this point to the `icon.png` in the current repo? I'd imagine not everyone wants the Quartz logo :)

Reply:
Same with the Twitter handle if possible! (maybe check if the links field in the config.toml has it)

https://github.com/jackyzha0/quartz/blob/dd11d56dd986c82914ce529fb43c8e2beb78a094/data/config.yaml#L22

Reply:
Yeah totally. Although I had trouble doing it relatively and deployed it on the server, I changed to an absolute path. But yeah, in theory, it should work relative as well. 

When I find some time, I can query the `config.toml`I , thought about that but took the lazy fast approach :)

Reply:
hi @sspaeti ! wondering if you ever got around to this :)

Reply:
I solved it, but could be more pretty. Ideas are welcome, see comment below :)